### DISCUSSION TITLE
is it possible to keep an overlay opened after a page refresh

### DISCUSSION THREAD

**Discussion by ammezie** [unpoly/unpoly#570] [Q&A]
Hi, is it possible to keep an overlay opened after a page refresh? I'm opening a link as an overlay using `up-layer="new"` but when the page is refreshed the overlay (the linked fragment) is opened on it own page instead of being stacked on the page it was triggered.

**Comment by triskweline** [comment]
In short: No.

Unpoly encourages all server routes (including those shown as overlays) to be prepared to render full HTML pages. In particular [subinteractions](https://unpoly.com/subinteractions) make it easy to implement interactions that work both on the root layer, and in an overlay.

If you absolutely cannot work with this, you you can choose to open an overlay without updating the history URL. Reloading will then show the root layer.

**Reply by ammezie** [reply]
Understandable. I'm new to unpoly and still trying to grasp the concepts.

> you can choose to open an overlay without updating the history URL

How can I go about this? Can you please point me to where to look in the docs.

**Reply by triskweline** [reply]
> How can I go about this? Can you please point me to where to look in the docs.

When opening an overlay with HTML attributes, you can set an `[up-history=false]` attribute.

For other ways see [History in overlays](https://unpoly.com/updating-history#overlays).
